xarray/backends/rasterio_.py

```diff
@@ -175,9 +175,7 @@ def open_rasterio(filename, parse_coordinates=None, chunks=None, cache=None, loc
     You can generate 2D coordinates from the file's attributes with::
 
         >>> from affine import Affine
-        >>> da = xr.open_rasterio(
-        ...     "https://github.com/mapbox/rasterio/raw/master/tests/data/RGB.byte.tif"
-        ... )
+        >>> da = xr.tutorial.open_rasterio("RGB.byte")
```

Comment:
I'm not really sure about this: we don't want to specify the url here, but this is a example for `open_rasterio` not `tutorial.open_rasterio`

Reply:
I think it's probably better to keep the original function since this is intended as a usage example. If we're concerned about the file potentially moving, we could add a tag, e.g., `https://github.com/mapbox/rasterio/raw/1.2.1/tests/data/RGB.byte.tif`